Collapsible/index.jsx

```diff
@@ -0,0 +1,62 @@
+import React, {Component} from 'react'
+import {Motion, spring} from 'react-motion'
+import debounce from '../lib/debounce'
+
+export default class Collapsible extends Component {
+  constructor () {
+    super()
+
+    this.state = {
+      height: 0
+    }
+  }
+
+  componentDidMount () {
+    this.debouncedResizeHandler = debounce(() => this.onResize())
+    window.addEventListener('resize', this.debouncedResizeHandler)
```

Comment:
What is the handling of resize events actually needed for ?
Does the height change for the element(s) on resize ?